resources/macos_user.rb

```diff
@@ -25,6 +26,14 @@ def sysadminctl
     '/usr/sbin/sysadminctl'
   end
 
+  def full_name?
```

Comment:
A method ending with a `?` in ruby is a signal that the method returns a boolean value. It's returning a string here, so we can remove the question mark.